client/README.md

```diff
@@ -193,8 +197,8 @@ log_format_json = true
 block_matrix_partition = "1/20"
 # Disables proof verification in general, if set to true, otherwise proof verification is performed. (default: false).
 disable_proof_verification = false
-# Disables fetching of cells from RPC, set to true if client expects cells to be available in DHT (default: false)
-disable_rpc = false
+# Network mode: 'both' uses P2P and RPC, 'p2p_only' disables RPC, 'rpc_only' disables P2P (default: both)
+network_mode = "both"
```

Comment:
This is breaking change, but assuming that nobody set `disable_rpc` to `true`, I guess we are ok.